Electric-Phantoms/welcome-to-nashville
# Sp fetchcalltry2

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


- [x ] New feature (non-breaking change which adds functionality)


# Testing Instructions

git fetch --all
git checkout sp-fetchcalltry2
run server (make sure dev tools are open)
search Fast Food in the second search bar (make sure you have the capital letters), then check console that an array of strings is there that looks like a list of fast food places
try the same thing to search Donuts (make sure you have the capital D)
look and see if you see anything else wrong 



# Checklist:

- [ x] My code follows the style guidelines of this project
- [x ] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [x ] I have added test instructions that prove my fix is effective or that my feature works
